vllm/utils.py

```diff
@@ -610,8 +618,10 @@ def get_kv_cache_torch_dtype(
                 raise ValueError(f"Invalid model dtype: {model_dtype}")
         elif cache_dtype in ["half", "bfloat16", "float"]:
             torch_dtype = STR_DTYPE_TO_TORCH_DTYPE[cache_dtype]
-        elif cache_dtype == "fp8":
-            torch_dtype = torch.uint8
+        elif cache_dtype == "fp8" or cache_dtype == "fp8_e4m3":
```

Comment:
```suggestion
        elif cache_dtype in ("fp8", "fp8_e4m3"):
```

Reply:
@comaniac don't we run into the same issue with kernels relying on the current uint8 type from https://github.com/vllm-project/vllm/pull/7302?

Reply:
Ah yeah sorry for missing that (apparently review PR with a small phone screen is not a good idea...) The torch_dtype should remain in uint8, but using `kv_cache = kv_cache.view` in FlashInfer backend should still work.